index.bs

```diff
@@ -2899,14 +2898,14 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
 
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialDescriptor">
     :   <dfn>type</dfn>
-    ::  This member contains the type of the [=public key credential=] the caller is referring to.
+    ::  This member contains the type of the [=public key credential=] the caller is referring to. The value SHOULD be a member of {{PublicKeyCredentialType}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the member was unset.
```

Comment:
Since this is a required member, treating it &ldquo;as if the member was unset&rdquo; would mean a parse error, which is what we're trying to avoid. How about &ldquo;ignoring any [[PublicKeyCredentialDescriptor]] with an unknown type&rdquo;.